tsdb/index/index.go

```diff
@@ -610,10 +651,10 @@ func (w *Writer) writeLabelIndices() error {
 	values := []uint32{}
 	for d.Err() == nil && cnt > 0 {
 		cnt--
-		d.Uvarint()               // Keycount.
-		name := d.UvarintBytes()  // Label name.
-		value := d.UvarintBytes() // Label value.
-		d.Uvarint64()             // Offset.
+		d.Uvarint()                           // Keycount.
+		name := d.UvarintBytes()              // Label name.
+		value := yoloString(d.UvarintBytes()) // Label value.
```

Comment:
#15455 changed this from `yoloString` to `string`. Looking the code over, I'm not able to see any reason for not using `yoloString`. Am I missing anything?

Reply:
It changed to `[]byte`, thus dropping a "yolo".  But you need a `string` for `ReverseLookup`.